hello_vsi/hello_vsi.csolution.yml

```diff
@@ -1,13 +1,28 @@
+# Copyright (c) 2024 Arm Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
 solution:
   created-by: uv2csolution@1.2.0
   created-for: CMSIS-Toolbox@2.0.0
-  compiler: AC6
-  cdefault: # use default setup of toolchain specific controls.
+  description: Hello example for the Video Streaming Interface
+
+  packs:
+    - pack: ARM::CMSIS@6.0.0
```

Comment:
Why would you specify multiple pack versions of the same pack concurrently. If the components are without version, the component version from the most recent selected pack will be selected. 
Take a look at the `hello_vsi.cbuild-pack.yml` file to see which pack versions are actually used.

Reply:
After a review with Vladimir the pack requirements have been moved to the hello_vsi.cproject.yml file.
Dedicasted component versions have been set for this project.